app/controllers/home_controller.rb

```diff
@@ -0,0 +1,4 @@
+class HomeController < ApplicationController
+  def home
```

Comment:
(@tomalhossain I recently moved to the dev team from the FED Team, so I'm picking some of this up along with you)